test/unittest/test_generateclasificationproject.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python
+
+# Copyright (C) 2006-2013  Music Technology Group - Universitat Pompeu Fabra
+#
+# This file is part of Gaia
+#
+# Gaia is free software: you can redistribute it and/or modify it under
+# the terms of the GNU Affero General Public License as published by the Free
+# Software Foundation (FSF), either version 3 of the License, or (at your
+# option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more
+# details.
+#
+# You should have received a copy of the Affero GNU General Public License
+# version 3 along with this program. If not, see http://www.gnu.org/licenses/
+
+
+import os
+import shutil
+import unittest
+
+import gaia2.fastyaml as yaml
+from gaia2.scripts.classification.generate_classification_project import generate_project
+
+
+class TestGenerateClassificationProject(unittest.TestCase):
+    def check_project(self, groundtruth_file, filelist_file, expected):
+        tmp_dir = 'tmp/'
```

Comment:
you can use `with tempfile.TemporaryDirectory`, and it'll clean up once it exits the with block